examples/run_smc_on_biphenyl.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from datetime import datetime
+from pickle import dump
+
+from timemachine.fe.absolute_hydration import set_up_ahfe_system_for_smc
+from timemachine.md.smc import sequential_monte_carlo
+from timemachine.testsystems.ligands import get_biphenyl
+
+
+def parse_options():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--n_walkers", type=int, help="number of walkers", default=100)
+    parser.add_argument("--n_windows", type=int, help="number of lambda windows", default=100)
+    parser.add_argument("--n_md_steps", type=int, help="number of MD steps per move", default=100)
+    parser.add_argument("--resample_thresh", type=float, help="resample when fractional ESS < thresh", default=0.6)
+    parser.add_argument("--debug_mode", type=bool, help="save full trajectories", default=False)
```

Comment:
should this accept a random seed argument? This seems at least useful for making `test_examples.py` deterministic.

Reply:
Would be useful -- added in https://github.com/proteneer/timemachine/pull/718/commits/ec5cf5c69cc393497b4a2e0c72fec0a69f4e8c1e